docs/en/differenze_oidc_fed.rst

```diff
@@ -0,0 +1,33 @@
+.. include:: ./common_definitions.rst
+
+
+.. _differenze_con_oidc_federation:
+
+Differences with the OIDC Federation
+------------------------------------
+
+This section lists the differences between the official standard and the SPID / CIE implementation.
+
+
+Client Registration
++++++++++++++++++++
+
+SPID supports only **automatic_client_registration**. The **implicit** mode is not supported.
+
+
+Listing endpoint
+++++++++++++++++
+
+In the SPID Federation, for this endpoint, the claim **entity_type** is added to all the `OIDC-FED`_ standard ones, in order to obtain a filter on the type of subordinate entities. This need apecifically allows filtering entities of type **federation_entity**, **openid_relying_party**, **openid_provider** and **oauth_resource**.
```

Comment:
standard ones?

Reply:
ho messo "is added to all the other `OIDC-FED`_ standard claims"